styles/Footer.module.scss

```diff
@@ -0,0 +1,315 @@
+    }
+
+    &Contact {
+        display: flex;
+        flex-direction: row;
+        max-width: 340px;
+    }
+
+    &ContactInformation {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    &EmailLink {
+        display: block;
+        line-height: 24px;
+        font-weight: $weight--bold;
+        color: var(--color--dark-indigo);
+        text-decoration-line: underline;
+
+        // This follows the designs as the @ doesn't cut through the line
+        text-underline-position: under;
+        // The extra spacing doesn't necessarily look as good.
+        // Maybe we should consider a pseudoelement underscore instead?
+    }
+
+    &PhoneLink {
+        display: block;
+        line-height: 24px;
+        font-weight: $weight--bold;
+        margin-top: 8px;
+    }
+
+    &Image {
+        max-width: 85px;
+        max-height: 85px;
+        border-radius: 85px;
+        overflow: hidden;
+        margin-right: 12px;
+    }
+}
+
+.secondaryContent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    @include media-query(large) {
+        flex-direction: row;
+        padding: $grid * 2 0 $grid $grid * 4;
+    }
+}
+
+.appendix {
+    display: flex;
+    flex-direction: row;
+    order: 2;
+
+    @include media-query(large) {
+        order: 1;
+        width: 50%;
+    }
+
+    &CopyrightNotice {
+        color: var(--color--grey);
+        @include font-size(xxs);
+        margin: 0;
+    }
+
+    &Link {
+        display: block;
+        margin-left: $grid;
+        color: var(--color--dark-indigo);
+        @include font-size(xxs);
+    }
+}
+
+.socialMediaIcons {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    order: 1;
+    margin-bottom: $grid;
+
+    @include media-query(large) {
+        order: 2;
+        margin-bottom: 0;
+        width: 50%;
+        transform: translateX(-20%);
+    }
+
+    &ListItem {
+        margin-left: $grid * 1.5;
+
+        &:first-child {
+            margin-left: 0;
+        }
+    }
+
+    &Link:focus path {
+        fill: var(--color--coral-dark);
+    }
+
+    &Icon:hover path {
+        fill: var(--color--coral-dark);
+    }
+}
+
+.email {
+    &Container {
+        background-color: var(--color--dark-indigo);
+        display: flex;
+        flex-direction: column;
+        padding: $grid;
+        text-align: center;
+
+        @include media-query(large) {
+            padding: $grid * 1.5 $grid * 2;
+            flex-direction: row;
+            align-items: center;
+        }
+
+        @include media-query(menu-break) {
+            padding: $grid * 1.5 $grid * 4;
+        }
+    }
+
+    &CTA {
+        color: var(--color--white);
+        @include font-size(ml);
+        margin-bottom: $grid * 0.75;
+
+        @include media-query(large) {
+            margin-bottom: 0;
+        }
+    }
+
+    &Label {
+        position: absolute;
+        top: 0;
+        left: -9999px;
```

Comment:
**Question:** Have you tried the `hidden` mixin?

Reply:
That's a good shout, thanks